coldfront/core/group/tests/management/test_create_default_user_groups.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management import call_command
+from django.test import TestCase
+from django.contrib.auth.models import Group, Permission
+from coldfront.core.group.management.commands.create_default_user_groups import (
+    DEFAULT_RIS_USER_GROUPS,
+)
+
+
+class CreateDefaultUserGroupsTest(TestCase):
+    """
+    Test the create_default_user_groups management command
+    """
+
+    def test_create_default_user_groups(self):
```

Comment:
@gunnarwustl This test appears to be a repetition of the code with some assertions. Would you consider for instance the follow?

- Test that a groups should have some permission
- Test that the app should have at least a group
- Is the default group designated by name or by a bool? In this case, I would test that designated default group exists.
- Test some of the negative tests such as what happens if there are no groups: expects to "do nothing" or throw an exception.

This was the expected behavior is described in the test.